astropy/samp/tests/test_standard_profile.py

```diff
@@ -31,9 +27,10 @@ def client_init_kwargs(self):
     def client_connect_kwargs(self):
         return {}
 
-    def setup_method(self, method):
+    @pytest.fixture(autouse=True)
+    def setup_method(self, tmp_path):
 
-        self.tmpdir = tempfile.mkdtemp()
+        self.tmpdir = str(tmp_path)
```

Comment:
Wouldn't it be better to use the `tmp_path` fixture in the tests that need it instead of defining this attribute?

Reply:
`samp` is moving to PyVO in the near future, refactoring it will just make the move harder if they started it already. Unless @tomdonaldson wants to refactor it here first? But I also don't have to a lot of time to spend on this.

Reply:
I agree refactoring this is not worth it if it complicates future plans.